packages/hooks/src/__tests__/useAvailableWallets.spec.tsx

```diff
@@ -42,6 +42,14 @@ jest.mock('@deriv/api', () => ({
 }));
 
 describe('useAvailableWallets', () => {
+    const wrapper = (mock: ReturnType<typeof mockStore>) => {
```

Comment:
hey hey @arshad-rao-deriv 
since here `wrapper` is a function that creates wrapper maybe we can use createWrapper name?: 
```
const createWrapper = (mock: ReturnType<typeof mockStore>) => {
    return ({ children }: { children: JSX.Element }) => (
        <APIProvider>
            <StoreProvider store={mock}>{children}</StoreProvider>
        </APIProvider>
    );
};
```
just suggestion, you can ignore it if anything

Reply:
Updated.